UPGRADE-4.0.md

```diff
@@ -228,6 +228,8 @@ Form
    ));
    ```
 
+ * Removed `ChoiceLoaderInterface` implementation in `TimezoneType`.
```

Comment:
I'm not sure we need this in the `UPGRADE.md` file. Unless there is a new code path to show.